src/__tests__/lib/containers/FacetsPlotNav.test.tsx

```diff
@@ -1,172 +0,0 @@
-import * as React from 'react'
```

Comment:
I think this was my original prototype for the facet plot-based navigation, was not referenced outside of a test.